docs/application/native/guides/notification/notification-ex.md

```diff
@@ -0,0 +1,532 @@
+    if (ret != NOTI_EX_ERROR_NONE)
+        /* Error handling */
+    ```
+
+- CheckBox Item:
+
+  This item can provide a user-selectable checkbox and can be toggled. In addition, the select state and clear state can be determined in advance.
+    ```
+    ret = noti_ex_item_checkbox_create(&checkbox_item, "checkbox_id", "checkbox_title", false);
+    if (ret != NOTI_EX_ERROR_NONE)
+        /* Error handling */
+    ```
+
+- Input Selector Item:
+
+  This item predefines an answer that you can choose from the notification for which a reply is requested.
+    ```
+    noti_ex_item_h selector_item = NULL;
+    ret = noti_ex_item_input_selector_create(&selector_item, "selector_id");
+    if (ret != NOTI_EX_ERROR_NONE)
+        /* Error handling */
+
+    const char *contents[] = {"What's up?", "On my way", "Nice"};
+    ret = noti_ex_item_input_selector_set_contents(selector_item, contents, 3);
+    if (ret != NOTI_EX_ERROR_NONE)
+        /* Error handling */
+    ```
+
+- Group Item:
+
+  This item can create a group using multiple items.
+    ```
+    noti_ex_item_h group_item = NULL;
+    noti_ex_item_h child_item = NULL;
+
+    ret = noti_ex_item_button_create(&child_item, "button_id", "button_title");
+
+    ret = noti_ex_item_group_create(&group_item, "group_id");
+    if (ret != NOTI_EX_ERROR_NONE)
+         /* Error handling */
+
+    ret = noti_ex_item_group_add_child(group_item, child_item);
+    if (ret != NOTI_EX_ERROR_NONE)
+        /* Error handling */
+    ```
+
+- Entry Item:
+
+  This item can be an input directly from the notification. This is necessary, to make a notification that you can reply to.
+    ```
+    noti_ex_item_h entry_item = NULL;
+
+    ret = noti_ex_item_entry_create(&entry_item, "entry_id");
+    if (ret != NOTI_EX_ERROR_NONE)
+        /* Error handling */
+
+    ret = noti_ex_item_entry_set_text(entry_item, "entry_text");
+    if (ret != NOTI_EX_ERROR_NONE)
+        /* Error handling */
+    ```
+
+- Progress Item:
+
+  This item allows you to create a progress bar and update the progress data.
+    ```
+    noti_ex_item_h progress_item = NULL;
+
+    ret = noti_ex_item_progress_create(&progress_item, "progress_id", 0, 50, 100);
+    if (ret != NOTI_EX_ERROR_NONE)
+        /* Error handling */
+
+    ret = noti_ex_item_progress_set_current(progress_item, 70);
+    if (ret != NOTI_EX_ERROR_NONE)
+        /* Error handling */
+    ```
+
+- Time Item:
+
+  This item contains time information.
+    ```
+    ret = noti_ex_item_time_create(&time_item, "time_id", time_info);
+    if (ret != NOTI_EX_ERROR_NONE)
+        /* Error handling */
+    ```
+
+<a name="group"></a>
+## Creating Group Item and add child
+
+- You can add an item as a child of a group using `noti_ex_item_group_add_child()`.
+
+  A group item means a series of items. Viewers can display them in the order they are added child. There can be multiple groups in a notification.
```

Comment:
@mk5004lee 
Thank you for the explanation.
How about rephrasing to:

A group item is a series of items. You can add items to a group.
If you add multiple items to the child, the items are displayed in the order that they are added. There can be multiple groups in a notification.

Please verify.